cypress/integration/seededFlows/articleFlows/playOrPauseAnimatedImages.spec.js

```diff
@@ -0,0 +1,109 @@
+describe('play or pause animated images', () => {
+  // Animated images are only identified async in the backend, and we can't rely on seeding them in the E2E DB
+  // Instead, we inject 3 images into the page body for testing purposes. 2 with the data-animated attribute, and 1 without.
```

Comment:
@nickytonline - I got there in the end 😄 My problem before is that I was trying to amend attributes on images in the seeded markdown (which weren't loaded yet in `before:load`) - injecting new DOM elements with the attributes seems to work just fine (although looks a little strange if you watch the test execution)